src/pages/Dashboard/Dashboard.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import Btn from 'components/Button'
+import Header from 'components/Header'
+import './Dashboard.module.css'
 
 export default function Dashboard ({ me, happStoreUser, registerUser }) {
-  return <div>
+  return <div styleName='container'>
+    <Header title='My HoloPort' />
 
-    <div>
-      <Link to='/menu' >Menu</Link>
+    <div styleName='linkBox'>
+      <h2><a href='/browse-happs'>Hosting</a></h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
     </div>
 
-    Here's the dashboard. It will have more information than this later.
-
-    <p>
-      User data: {me && JSON.stringify(me)}
-    </p>
+    <div styleName='linkBox'>
+      <h2><a href='/earnings'>Earnings</a></h2>
```

Comment:
can you use react router links for these? There's a major difference in performance

Reply:
Oh yeah, duh. Done.